video_source: '/dev/video0'

camera_matrix: '[[1204.4697584542726, 0.0, 354.6182298916754], [0.0, 1201.8571173383637,
  652.7026112972777], [0.0, 0.0, 1.0]]'
dist_coeffs: '[[0.008252427738841408, 0.17962144633096283, -0.0010671121861827175,
  -0.0017195153485555525, -0.7083074964169391]]'
marker_size: 0.075
marker_spacing: 0.5
markers_x: 7
markers_y: 7
x_offset: 0
y_offset: -0.10
z_offset: 0.12
